src/api/scenarioData/scenarioData.service.ts

```diff
@@ -288,9 +301,9 @@ export class ScenarioDataService {
         return scenarios;
 
       case 'number-input':
-        const numberValues = validationCriteria?.split(',').map((val: string) => val.trim());
+        const numberValues = validationCriteria?.split(',').map((val: string) => this.cleanValue(val));
         const minValue = (numberValues && parseInt(numberValues[0], 10)) || 0;
-
+        console.log(numberValues, 'number values');
```

Comment:
Console log left in (two more further below as well)

Reply:
Thank you! Forgot cleanup.